README.md

```diff
@@ -46,6 +46,40 @@ SKIP_AUTH=1 FORMS_ADMIN_URL='http://localhost:3000/' PRODUCT_PAGES_URL='http://l
 The end to end tests can be run without visiting the product pages by setting
 the `SKIP_PRODUCT_PAGES` environment variable to `1`.
 
+### Skipping the s3 submission test
+
+The end to end tests can be run without testing a form with the `submission_type` of `s3` by setting the `SKIP_S3` environment variable to `1`.
+
+### Running the s3 submission test
+
+- in `forms-runner`: 
+    - add your govuk_notify.api_key and iam_role_arn (this will be the ARN of the role with permissions to upload to the s3 bucket in the deploy environment) to the settings.yml
```

Comment:
Also, I'm wondering if we should encourage developers to use environment variables rather than the settings files, to reduce the chance of having secrets in plaintext in the repo.

We should at least say `settings.local.yml` instead of `settings.yml`, so things don't get accidentally committed:

```suggestion
    - add your govuk_notify.api_key and iam_role_arn (this will be the ARN of the role with permissions to upload to the s3 bucket in the deploy environment) to the settings.local.yml
```

Reply:
One more thing, is there any documentation we can link to about this IAM role and how to find its ARN?

Reply:
Hmm, I think a link to the code is useful, things do change but it's better than nothing. I guess we could use a GitHub permalink so if the code does change we'll at least know what the link was supposed to point at.